src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/CountFalsePositivesTest.java

```diff
@@ -24,20 +26,18 @@
      */
     @Test
     public void testEvaluateMutect2() throws Exception {
-        final String dreamDir =  publicTestDir + "org/broadinstitute/hellbender/tools/mutect/dream/vcfs/";
-        List<String> vcfList = Arrays.asList(dreamDir + "sample_1.vcf", dreamDir + "sample_2.vcf", dreamDir + "sample_3.vcf", dreamDir + "sample_4.vcf");
-        final File vcfListFile = createTempFile("vcfList", ".txt");
+        final String dreamDir =  publicTestDir + "org/broadinstitute/hellbender/tools/mutect/dream/";
         final File output = createTempFile("output", ".txt");
-        Path file = Paths.get(vcfListFile.getAbsolutePath());
-        Files.write(file, vcfList, Charset.forName("UTF-8"));
         final String[] args = {
-                "-V", vcfListFile.toString(),
+                "-V", dreamDir + "vcfs/dream3-chr20.vcf",
```

Comment:
`args` is a string array though so with a file I'd have to call `toString()`. Figured it's simpler to work directly with a string.